cfg/MatchZy/live_wingman.cfg

```diff
@@ -103,4 +103,4 @@ sv_voiceenable 1
 tv_relayvoice 0
 
 // execute override config
-exec live_wingman_override.cfg
\ No newline at end of file
+exec MatchZy/live_wingman_override.cfg
```

Comment:
Can you please make the same change in live.cfg as well?
Thanks!

Reply:
Done. I think I resynced to main and undid the initial change.